### PR TITLE
psp-6988 File tabs fix

### PR DESCRIPTION
**Pull request by devinleighsmith**

### CODE REVIEW

File: .github/workflows/ci-cd-pims-dev.yml [hunk 1]
```diff
@@ -123,7 +123,7 @@ jobs:
       - name: call scripts to upgrade database
         shell: bash
         run: |
-          oc process -f ./openshift/4.0/templates/jobs/db-deploy.yaml -p SERVER_NAME=sqldevtst.th.gov.bc.ca -p DB_NAME=PIMS_DEV -p NAMESPACE=3cd915-dev | oc create -f - | grep -oP "(?<=job\.batch/)[^\s]*" | (read JOB_NAME; oc wait --for=condition=complete job/$JOB_NAME --timeout=120s)
+          oc process -f ./openshift/4.0/templates/jobs/db-deploy.yaml -p DB_SECRET_NAME=pims-database GIT_BRANCH=dev SERVER_NAME=sqldevtst.th.gov.bc.ca -p DB_NAME=PIMS_DEV -p NAMESPACE=3cd915-dev | oc create -f - | grep -oP "(?<=job\.batch/)[^\s]*" | (read JOB_NAME; oc wait --for=condition=complete job/$JOB_NAME --timeout=120s)
```
Comment (asanchezr) [inline .github/workflows/ci-cd-pims-dev.yml:126]: this doesn't seem related to that story but I guess we can fix it anyways?

File: openshift/4.0/templates/jobs/db-deploy.yaml [hunk 2]
```diff
@@ -65,3 +65,9 @@ parameters:
   - name: NAMESPACE
     description: the openshift namespace to run this job in.
     required: true
+  - name: GIT_BRANCH
+    description: the git branch to pull the database scripts from.
+    required: true
+  - name: DB_SECRET_NAME
+    description: the git branch to pull the database scripts from.
+    required: true
```
Comment (asanchezr) [inline openshift/4.0/templates/jobs/db-deploy.yaml:73]: Question - I'm assuming the corresponding secrets have been updated/configured in OCP?

File: .github/workflows/ci-cd-pims-dev.yml (hunk 1 above)
Comment (devinleighsmith) [reply .github/workflows/ci-cd-pims-dev.yml]: This was from https://github.com/bcgov/PSP/pull/3573, removed from this PR.

**Review by asanchezr — APPROVED** [review]
